### ISSUE TITLE
ui5-multi-combobox: keyboard navigation over the tokens in ltr mode

### ISSUE THREAD

**Issue by unazko** [SAP/ui5-webcomponents#11826]
### Bug Description

The keyboard navigation over the tokens doesn't work properly when the page is in right to left mode.

### Affected Component

ui5-multi-combobox

### Expected Behaviour

I would expect that the arrow right would do what arrow left does in `rtl` mode.

### Isolated Example

https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpOyBoZWlnaHQ6IDM1MHB4O1wiPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cblxuICA8ZGl2IGRpcj1cInJ0bFwiPiAgXG4gIDx1aTUtbXVsdGktY29tYm9ib3ggcGxhY2Vob2xkZXI9XCJUeXBlIHlvdXIgdmFsdWVcIj5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiQWxiYW5pYVwiPjwvdWk1LW1jYi1pdGVtPlxuICAgICAgICA8dWk1LW1jYi1pdGVtIHNlbGVjdGVkIHRleHQ9XCJBcmdlbnRpbmFcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiQnVsZ2FyaWFcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiRGVubWFya1wiPjwvdWk1LW1jYi1pdGVtPlxuICAgICAgICA8dWk1LW1jYi1pdGVtIHRleHQ9XCJFbmdsYW5kXCI-PC91aTUtbWNiLWl0ZW0-XG4gICAgICAgIDx1aTUtbWNiLWl0ZW0gdGV4dD1cIkdlcm1hbnlcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiUGhpbGlwcGluZXNcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiUG9ydHVnYWxcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiVGhlIFVuaXRlZCBLaW5nZG9tIG9mIEdyZWF0IEJyaXRhaW4gYW5kIE5vcnRoZXJuIElyZWxhbmRcIj48L3VpNS1tY2ItaXRlbT5cbiAgICA8L3VpNS1tdWx0aS1jb21ib2JveD5cbiAgPC9kaXY-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9NdWx0aUNvbWJvQm94LmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9NdWx0aUNvbWJvQm94SXRlbS5qc1wiOyJ9fQ

### Steps to Reproduce

1. Open the [sample](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpOyBoZWlnaHQ6IDM1MHB4O1wiPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cblxuICA8ZGl2IGRpcj1cInJ0bFwiPiAgXG4gIDx1aTUtbXVsdGktY29tYm9ib3ggcGxhY2Vob2xkZXI9XCJUeXBlIHlvdXIgdmFsdWVcIj5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiQWxiYW5pYVwiPjwvdWk1LW1jYi1pdGVtPlxuICAgICAgICA8dWk1LW1jYi1pdGVtIHNlbGVjdGVkIHRleHQ9XCJBcmdlbnRpbmFcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiQnVsZ2FyaWFcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiRGVubWFya1wiPjwvdWk1LW1jYi1pdGVtPlxuICAgICAgICA8dWk1LW1jYi1pdGVtIHRleHQ9XCJFbmdsYW5kXCI-PC91aTUtbWNiLWl0ZW0-XG4gICAgICAgIDx1aTUtbWNiLWl0ZW0gdGV4dD1cIkdlcm1hbnlcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiUGhpbGlwcGluZXNcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiUG9ydHVnYWxcIj48L3VpNS1tY2ItaXRlbT5cbiAgICAgICAgPHVpNS1tY2ItaXRlbSB0ZXh0PVwiVGhlIFVuaXRlZCBLaW5nZG9tIG9mIEdyZWF0IEJyaXRhaW4gYW5kIE5vcnRoZXJuIElyZWxhbmRcIj48L3VpNS1tY2ItaXRlbT5cbiAgICA8L3VpNS1tdWx0aS1jb21ib2JveD5cbiAgPC9kaXY-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9NdWx0aUNvbWJvQm94LmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9NdWx0aUNvbWJvQm94SXRlbS5qc1wiOyJ9fQ)
2. Try to navigate with the right arrow key trough the tokens

_Observe_:
No navigation is performed as a result of a right arrow key press

3. Navigation trough the tokens via left arrow key

_Observe_:
The focus gets stuck at some point and no backward navigation is possible via right arrow key.

### Priority

None

### UI5 Web Components Version

latest

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

Here is an issue with the `ui5-multi-comobox` component.

Best regards,
Boyan
